### DISCUSSION TITLE
Scan roms inside archives?

### DISCUSSION THREAD

**Discussion by netwaretcs** [emmercm/igir#556] [Q&A] [answered]
Can Igir scan roms within a zip archive of the entire system? 
E.g. snes.zip / c64.zip...

**Comment by emmercm** [comment, marked as answer]
It can, yep. The size of the archive shouldn't matter, but there are some such as gzip that are a little less efficient (see the [docs](https://igir.io/archives/#supported-archive-types-for-reading)).

The only thing `igir` doesn't support is archives within archives, as that would be fairly expensive to parse.
